README.rst

```diff
@@ -100,12 +100,30 @@ This will add these paths to Django:
 
 You can use them like this in your django templates:
 
+- GET (deprecated):
+
 .. code-block:: html
 
     <a href="{% url 'django_auth_adfs:logout' %}">Logout</a>
     <a href="{% url 'django_auth_adfs:login' %}">Login</a>
     <a href="{% url 'django_auth_adfs:login-no-sso' %}">Login (no SSO)</a>
 
+- POST (recommended):
+
+.. code-block:: html
+
+    <form method="post" action="{% url 'django_auth_adfs:logout' %}">{% csrf_token %}
+        <button type="submit">Logout</button>
+    </form>
+    <form method="post" action="{% url 'django_auth_adfs:login' %}">{% csrf_token %}
+        <input type="hidden" name="next" value="{{ next }}">
+        <button type="submit">Login</button>
+    </form>
+    <form method="post" action="{% url 'django_auth_adfs:login-no-sso' %}">{% csrf_token %}
+        <input type="hidden" name="next" value="{{ next }}">
+        <button type="submit">Login (no SSO)</button>
+    </form>
```

Comment:
What do you think about adding the following form example? It's something I had to do for another project and it's limited to requiring javascript.

```django
{# Requires javascript, but avoids the button element #}
<form method="post" action="{% url 'django_auth_adfs:logout' %}">{% csrf_token %}
    <a href="#" onclick="this.closest('form').submit()">Logout</a>
</form>
```
```

Reply:
Thanks for the suggestion! I see the advantages of using the link approach, especially for styling purposes. However, I’m leaning towards using the standard <button> element for a few reasons:

- **CSP Compatibility**: Inline JavaScript might conflict with strict Content Security Policy (CSP) settings.

- **URL Issue**: The **href="#"** adds **#** to the URL, which could create unnecessary entries in the browser history and potentially confuse users.

That said, I’m happy to include both methods in the documentation if you’re okay with it.

Reply:
Nope, that's good enough for me. Thanks for the feedback!